corehq/apps/app_manager/suite_xml/sections/details.py

```diff
@@ -167,20 +188,25 @@ def build_detail(self, module, detail_type, detail, detail_column_infos, tabs=No
                 #   column_info.column: an instance of app_manager.models.DetailColumn
                 #   column_info.sort_element: an instance of app_manager.models.SortElement
                 #   column_info.order: an integer
+                if "search" in id and column_info.sort_element and column_info.sort_element.type == 'index':
```

Comment:
`if "search" in id` seems fragile - is it possible to keep track of search detail ids somewhere rather than pattern matching on the id?

Reply:
I do see that we're already doing `detail_type.endswith('short')` elsewhere, which is the same kind of pattern, but still, might be better not to replicate that.

Reply:
good point. I've updated this PR so that the search detail is a "first class" detail, but I've still followed the `endswith` pattern, because that's the way all of these are checked in this code.